functions/Get-DbaPolicy.ps1

```diff
@@ -0,0 +1,100 @@
+﻿Function Get-DbaPolicy {
```

Comment:
This one will need to be renamed to `Get-DbaSqlPolicyManagement` unless @potatoqualitee has another preference. This is the noun we have for the Copy command.

Reply:
I was wondering about this because the Copy copies over the whole suite but this is just the policy and returns `Microsoft.SqlServer.Management.Dmf.Policy` So i think we'll keep this name